docs/source/cli.rst

```diff
@@ -139,5 +139,32 @@ This command reads the waveform definitions from the `YAML` file, evaluates them
 .. note::
     You must provide exactly one of `--linspace` or `--csv` for this command.
 
+export-xml
```

Comment:
Please name the PCSSP export as such (XML export is too generic) :)

Also holds for the code, classes, etc.